commit 23d43090e0b275e47e09e859823e965a1eb323dc
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   Thu Feb 26 02:46:02 2015 -0300

    Fix constness error just introduced in findlocale.
    
    for  ChangeLog
    
    	[BZ #15969]
    	* locale/findlocale.c (_nl_find_locale): Fix constness error in
    	the previous change.

diff --git a/locale/findlocale.c b/locale/findlocale.c
index 360f58b..5e2639b 100644
--- a/locale/findlocale.c
+++ b/locale/findlocale.c
@@ -161,7 +161,8 @@ _nl_find_locale (const char *locale_path, size_t locale_path_len,
       loc_name = (char *) _nl_expand_alias (*name);
       if (loc_name != NULL)
 	{
-	  data = _nl_load_locale_from_archive (category, &loc_name);
+	  data = _nl_load_locale_from_archive (category,
+					       (const char **) &loc_name);
 	  if (__builtin_expect (data != NULL, 1))
 	    return data;
 	}
